### ISSUE TITLE
workspace not loading properly

### ISSUE THREAD

**Issue by elcid82** [jaszhix/nmsde#3]
**Issue**
After starting NMSDE (0.1.1 AND 0.2.0) the workspace stays white (I saw in the other issue-thread, that it may be grey, not sure if it is just a Windows setting). Trying to open a .pak file does nothing. The workspace stays white.

**Steps to reproduce**
I simply started NMSDE both with and without admin rights.

**Other information**
My system specs:
AMD Phenom II X6 1090T  3.20 Ghz Processor
32 GB RAM
Win7 Ultimate 64bit (language set to german)
GeForce GTX960

DxDiag:
[DxDiag.txt](https://github.com/jaszhix/nmsde/files/1254370/DxDiag.txt)

Pics of NMSDE:
![nmsde_1](https://user-images.githubusercontent.com/31385244/29749655-5f854d0a-8b31-11e7-94c1-589c543ec00a.jpg)
![nmsde_2](https://user-images.githubusercontent.com/31385244/29749656-5f862d1a-8b31-11e7-8f4a-fa59329ff2dc.jpg)

If you need more info, just ask. I'll gladly try to help as best as I can!

**Comment by jaszhix** [comment]
Yes, its not loading the rest of the UI. I've always had lots of issues with Electron in Windows 7, and I'm not sure what's causing this. ~~Can you supply the nmsde.log file in AppData/Roaming/NMSDE?~~

Just saw your message on Disqus. If its not creating a log file, then could I walk you through launching the dev version of the app and copying the output in devtools?

To do so

* Download and install [Nodejs](https://nodejs.org/en/) and [yarn](https://yarnpkg.com/en/docs/install).
* Download the ZIP (or git clone if you use git) of the repository. Extract it, in the root of the directory type ```yarn```, and then type ```npm run dev```.
* After a bit the app should launch with devtools open. I am interested in any errors in red, or anything that looks like a stack trace.

Thanks!

**Comment by elcid82** [comment]
Ok, I did what you said. Downloaded and installed Node.js and yarn. Downloaded the zip-file and extracted it. Opened the command prompt of windows and changed to the nmsde-master folder. Typed yarn and waited for the prompt to respond, gave me a "success" so I typed npm run dev which gave me some red lines shown in the image below:
![nmsde-master-yarn npm-run](https://user-images.githubusercontent.com/31385244/29760248-2d86da8a-8bc2-11e7-886f-20534d06a14b.jpg)

So I checked the folder again (since it said that it was missing a file) and indeed, there was no package.json file in the nmsde-master folder. But I found another nmsde-master folder inside the first nmsde-master folder, so I checked this one out and there I found the package.json.

Back to the cmd prompt, I tried running npm rund dev again but this time in the second nmsde-master folder. There I got another error message, telling me that this time the "node_modules" was missing:
![nmsde-master-yarn npm-2nd_run](https://user-images.githubusercontent.com/31385244/29760344-f93d567c-8bc2-11e7-850f-63b49c491178.jpg)

So I copied the node_modules folder from the first nmsde-master to the second one and ran npm run dev again with the following output:
![nmsde-master-yarn npm-3rd_run](https://user-images.githubusercontent.com/31385244/29760394-8322f518-8bc3-11e7-88ba-b0b7ef7a8d53.jpg)

In both the second and third run pictures you can see some german text, saying that the command "cross-env" is either written wrong or couldn't be found.

Either way, NMSDE could not be started.

I'll add the mentioned log files for all three runs, maybe they will help you. (I just added .txt to the log files since .log isn't supported here.)
Log of the first run:
[2017-08-28T05_19_08_204Z-debug.log.txt](https://github.com/jaszhix/nmsde/files/1255210/2017-08-28T05_19_08_204Z-debug.log.txt)

Log of the second run:
[2017-08-28T05_31_09_478Z-debug.log.txt](https://github.com/jaszhix/nmsde/files/1255211/2017-08-28T05_31_09_478Z-debug.log.txt)

Log of the third run:
[2017-08-28T05_34_48_544Z-debug.log.txt](https://github.com/jaszhix/nmsde/files/1255212/2017-08-28T05_34_48_544Z-debug.log.txt)

**Comment by jaszhix** [comment]
Delete node_nodules, and run ```yarn``` inside nmsde-master. Then run ```npm run dev```. Never copy node_modules, if it messes up just regenerate them.

Edit: On second glance, its clear why it failed, package.json was removed from nmsde-master somehow. Its the file that manages all the npm scripts, so it needs to be in the same directory as node_modules. I'm guessing the way you extracted it caused the nmsde-master folder to get nested inside the actual one.

**Comment by elcid82** [comment]
Oh my... what a stupid mistake :D 
I'll try the 0.3.0 version first, then I'll try running yarn and the other stuff again if 0.3.0 doesn't work for me. I'll get back to you once I have some results.

EDIT:
OK, here you go. I hope this time I did everything right... Using version 0.3.0 gave me the same error, missing UI / workspace staying white.
So I downloaded the zip from the repository, this time I extracted it the right way (hopefully :D).
Then I ran yarn inside of the nmsde-master folder and it took way longer than last time. this was the output:
![nmsde-yarn output](https://user-images.githubusercontent.com/31385244/29836399-707543c0-8cf5-11e7-99ea-f69fbe3d6225.jpg)

Then I ran npm run dev and I got this:
![nmsde-npm_run_dev output](https://user-images.githubusercontent.com/31385244/29836426-8245a2ca-8cf5-11e7-9f78-f2af76a5dd9c.jpg)

NMSDE was still not opening (not sure if it should, though), instead I got 3 new log files which I attached below:
[2017-08-29T17_55_30_830Z-debug.log.txt](https://github.com/jaszhix/nmsde/files/1261218/2017-08-29T17_55_30_830Z-debug.log.txt)
[2017-08-29T17_55_31_725Z-debug.log.txt](https://github.com/jaszhix/nmsde/files/1261219/2017-08-29T17_55_31_725Z-debug.log.txt)
[2017-08-29T17_55_31_803Z-debug.log.txt](https://github.com/jaszhix/nmsde/files/1261217/2017-08-29T17_55_31_803Z-debug.log.txt)

I hope this time I did it right... sry, I'm a slightly advanced computer user, but no programmer, so I have no idea what I'm doing here ;)

EDIT2:
Just out of curiosity I simply ran npm run dev again without changing anything and this time it worked. Well... sort of. I didn't get any red errors and I have no clue what "stack traces" are. That's how it looked like:

![nmsde-npm_run_dev output 2nd try - console view](https://user-images.githubusercontent.com/31385244/29837701-2dcdb8aa-8cf9-11e7-9d97-1c1fe5485d4e.jpg)


I tried opening some .pak files, but NMSDE always told me that it couldn't find any mbins to decompile even though I know that there were mbins inside the pak files.
